deployments/Dockerfiles/documentation/Dockerfile

```diff
@@ -12,16 +12,13 @@ RUN apt-get install -y                                                       \
 ENV LC_ALL C.UTF-8
 ENV LANG C.UTF-8
 
-RUN python3 -m pip install -U pipenv
-
 RUN mkdir /build
 WORKDIR /build
+
 COPY docs/ /build/docs
 COPY mkdocs.yml /build
-COPY Pipfile* /build
-
-RUN pipenv install --dev --skip-lock --system
 
+RUN pip3 install bs4==0.0.1 markdown==3.3.4 click==8.0.2 mkdocs==1.3.0 markdown-include==0.6.0 mkdocs-material==7.1.10 mkdocs-macros-plugin==0.7.0 pymdown-extensions==8.2 Pygments==2.11.2
```

Comment:
Can we not have an updated Pipfile in the same folder. I think that's preferable.